server/routes/login/login.controller.js

```diff
@@ -3,6 +3,8 @@
  * GET    /    ->    index
  */
 
+const { isCourtUser } = require('../../components/auth/user-type.js');
+
```

Comment:
Doesnt seem to be used here? Should remove?

Reply:
Good spot, I was fiddling with where this should live.